test/src/device-request-prompt.spec.ts

```diff
@@ -19,6 +19,7 @@ import {TimeoutError} from 'puppeteer';
 import {launch} from './mocha-utils.js';
 
 describe('device request prompt', function () {
+  this.timeout(1_000);
   let state: Awaited<ReturnType<typeof launch>>;
 
   before(async () => {
```

Comment:
One consequence I did not anticipate is that it now the spawning of the Browser needs to happen in 1s which is too fast for `windows` so let's add a separate `this.timeout(10_000)` in the `before` hooks.

Reply:
done in https://github.com/puppeteer/puppeteer/pull/11185/commits/e7cadfe88180e6229c6fd315c73e08cebc0f68b5